### PR TITLE
NSIS Package installs i18n files

### LINKED ISSUE #335: Windows/Msys2 - i18n files not installed
This is I think due to the way we do translations on Windows/Msys2. There is no `install()` in the `CMakeLists.txt files`. They should end up in the `$Install_Dir/share/locale` directory at package creation time and populate the `translations` component.

This issue should cover two tasks

- [ ] Get the i18n files to install for Windows/Msys2
- [ ] The files should be copied and compiled as part of the translations component for the NSIS installer

### PR DESCRIPTION
**Pull request by jwrober**
Small update to get the NSIS package to contain the locale files. Fixes #335

### CODE REVIEW

File: cmake/GettextTranslate.cmake [hunk 1]
```diff
@@ -288,6 +288,7 @@ macro(GettextTranslate)
     install(FILES ${GMO_FILE_NAME} DESTINATION
       ${LOCALEDIR}/${lang}/LC_MESSAGES
       RENAME ${MAKEVAR_DOMAIN}.mo
+	  COMPONENT translations
```
Comment (lmoureaux) [inline cmake/GettextTranslate.cmake:291]: Tab :stuck_out_tongue_closed_eyes:

**Review by lmoureaux — CHANGES_REQUESTED** [review]
Looking good! I hate tabs because Github doesn't agree with your editor on how long they should be :smile:

File: cmake/GettextTranslate.cmake (hunk 1 above)
Comment (jwrober) [reply cmake/GettextTranslate.cmake]: Fixed. Agree I gotta figure out my editor on Windows.

**Review by lmoureaux — APPROVED** [review]
Can merge when the pipeline succeeds

**Pull request merged**